apps/api/src/utils/coin.ts

```diff
@@ -1,48 +1,51 @@
 import { asset_lists } from "@chain-registry/assets";
-import * as Sentry from "@sentry/node";
 import { Coin } from "cosmjs-types/cosmos/base/v1beta1/coin";
 
 export function coinToAsset(coin: Coin) {
-  try {
-    if (coin.denom === "uakt") {
-      return {
-        symbol: "AKT",
-        logoUrl: "https://console.akash.network/images/akash-logo.svg",
-        amount: parseInt(coin.amount) / 1_000_000
-      };
-    } else if (coin.denom === "akt") {
+  if (coin.denom === "uakt") {
+    return {
+      symbol: "AKT",
+      logoUrl: "https://console.akash.network/images/akash-logo.svg",
+      amount: parseInt(coin.amount) / 1_000_000
+    };
+  } else if (coin.denom === "akt") {
+    return {
+      symbol: "AKT",
+      logoUrl: "https://console.akash.network/images/akash-logo.svg",
+      amount: parseFloat(coin.amount)
+    };
+  } else {
+    const akashChain = asset_lists.find(c => c.chain_name === "akash");
+    const ibcAsset = akashChain.assets.find(a => a.base === coin.denom);
+
+    if (!ibcAsset) {
+      console.log(`Unknown asset ${coin.denom}`);
```

Comment:
do we need to act on this one somehow?

Reply:
Nothing to act on. I left a log so that if in the future we change our mind and want to see which one we are missing we can always filter the logs to find them.

Reply:
Also, perhaps log with the new pino logger?